docs/examples/pipelinerun_image.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: steward.sap.com/v1alpha1
+kind: PipelineRun
+metadata:
+  generateName: image-
+spec:
+  image: stewardci/stewardci-jenkinsfile-runner:TEST_201006_b4e6c67
```

Comment:
Consider adapting configurable JFR image and image pull policy

Reply:
Policy is now also configurable